packages/clone-testing/Cargo.toml

```diff
@@ -27,8 +32,9 @@ tonic = { workspace = true }
 [dev-dependencies]
 cosmwasm-schema = "2.0.0"
 speculoos = { workspace = true }
-cw20 = { workspace = true }
-cw20-base = { workspace = true }
+
+cw20 = { version = "2.0.0" }
```

Comment:
return this to `workspace = true`?

Reply:
No, there is a version mismatch with cosmwasm inside abstract-cw20 repo. We'll need to change that as soon as the new interfaces are in (and potentially inside this repo)